eng/validation/build-arcadewithrepo.ps1

```diff
@@ -210,6 +217,34 @@ function Git-Command($repoName) {
     }
 }
 
+function Get-SubscribedBranch()
```

Comment:
Is this actually used since the subscribed branch name looks to be passed explicitly?

Reply:
So, it was used. Then, yesterday, I noticed there were two branches subscribed to Latest for both of the repos we're using in this process, so the parameter we pass in was to override the look-up. I'm on the fence about which is better, as consuming repos could subscribe to Latest with a different branch name without us knowing, and then we'd have to do a PR to change the yaml; however, explicitly passing in the branch name allows us to still be able to do the expected look-ups of the LKG and builds as expected, without us having to worry about there being more than one branch subscribed to the Latest channel.